securedrop_client/gui/auth/dialog.py

```diff
@@ -51,6 +52,7 @@ class LoginDialog(QDialog):
     MIN_PASSWORD_LEN = 14  # Journalist.MIN_PASSWORD_LEN on server
     MAX_PASSWORD_LEN = 128  # Journalist.MAX_PASSWORD_LEN on server
     MIN_JOURNALIST_USERNAME = 3  # Journalist.MIN_USERNAME_LEN on server
+    PASSPHRASE_LABEL_SPACING = 0.5
```

Comment:
This one doesn't seem to be used (anymore?).

Reply:
You were right, I had completely forgotten to remove it after going with a different solution! I am surprised that all the tests passed while this one was still there unused.